src/vm/macroasm.rs

```diff
@@ -23,6 +23,8 @@
 #[macro_export]
 macro_rules! rgbasm {
     ($( $tt:tt )+) => {{ #[allow(unused_imports)] {
+        use amplify::num::u5;
```

Comment:
I found a minor issue here. 

```suggestion
        use amplify::num::{ u4, u5 };
```